### PR TITLE
Last minute fixes

### PR DESCRIPTION
**Pull request by Strate-S**
Typos and post-genocide interactions + anything else I might find along the way

### CODE REVIEW

File: Gaijinizer patch/Map037.txt [hunk 1]
```diff
@@ -833,8 +855,10 @@ No need to worry, I created them all anyways.
 
 こっちもさんざん面目潰されてキレてるんだぁ。
 > CONTEXT: Map037/events/4/pages/1/311/Dialogue/enemy3
-Do it all. I'm sorry that this is a very funny
-thing.#MTLed
+Try it.
+
+This one's face has also been completely cut and
+crushed.#StrateTemporary
```
Comment (contingency40) [inline Gaijinizer patch/Map037.txt:861]: I checked with a native speaker, and it appears she's saying something to the effect of.

"Give it a try.

I'm pretty pissed off too about being humiliated like that."

I'm assuming referring to the Barghest taking Kazuya's side.

File: Gaijinizer patch/Map037.txt [hunk 2]
```diff
@@ -36,62 +41,70 @@ I am Hecate. Barghest and Lampas's "owner"?#MTLed
...
 『遊び』………だと！？
 
 この惨劇が、全部『遊び』だっていうのか……！？
 > CONTEXT: Map037/events/4/pages/0/43/Dialogue/kazuya
-'Play'...that's it! ? Does this tragedy all
-mean "play"? ?#MTLed
+"Play"......!?
+
+This whole tragedy is just "play" to you......!?#StrateLocalization
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map037.txt:69]: Minor revisions for flow:
```
"Play"......!?

This whole tragedy is just playing to you..!?#StrateLocalization
```

File: Gaijinizer patch/Armors.txt [hunk 3]
```diff
@@ -284,66 +284,71 @@ Obedience Bracelet#Strate
 > BEGIN STRING
 豪華に装飾された、至高のもっふもふローブ
 魔法に関しての属性への耐性と、魔法・敏捷上昇の効果がある
-> CONTEXT: Armors/35/description < UNTRANSLATED
-
+> CONTEXT: Armors/35/description
+Lavishly decorated, supremely fluffy robe.
+Resists magic attributes and increases magic and agility.#Strate
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Armors.txt:290]: minor revision (magic attributes sounds clunky)
```
Lavishly decorated, supremely fluffy robe.
Resists magic damage while increasing magic and agility.
```

File: Gaijinizer patch/Armors.txt [hunk 4]
```diff
@@ -284,66 +284,71 @@ Obedience Bracelet#Strate
...
+Bleeding, and Soaked. Halves the probability of Weakness.#StrateUnsure
 > END STRING
 
 > BEGIN STRING
 リア・マーク・プロト
-> CONTEXT: Armors/37/name < UNTRANSLATED
-
+> CONTEXT: Armors/37/name
+Rear Mark Proto#StrateUnsure
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Armors.txt:324]: Note

File: Gaijinizer patch/Skills.txt [hunk 5]
```diff
@@ -6461,7 +6461,7 @@ damage five times, small chance to inflict "Ecstacy".#Ron
 > BEGIN STRING
 種搾りプレス
 > CONTEXT: Skills/522/name
-Reverse Piledriver#afrgPR
+Seed-squeezing Press#afrgPRStrateRevision
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Skills.txt:6465]: Capitalize the S here.

File: Gaijinizer patch/Troops.txt [hunk 6]
```diff
@@ -6652,8 +6651,10 @@ Arachne is with his hand nails. I threw my throat down
 > CONTEXT: Troops/200/pages/2/69/Dialogue/enemy10
 > CONTEXT: Troops/201/pages/2/69/Dialogue/enemy10
 > CONTEXT: Troops/212/pages/2/85/Dialogue/enemy10
-I'm going through the nest of alive and survive
-... I'll get the appropriate "karma" back ......#MTLed
+You will escape from my nest, and survive......
+
+But I'll have you carry the burden of that
+"karma"...#StrateUnsureLocalization
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Troops.txt:6658]: Note

File: Gaijinizer patch/Map132.txt [hunk 7]
```diff
@@ -126,8 +126,8 @@ Hy-Hyakki?!#Ron
 だがそれは瞬く間にティタノボアの蛇体に包まれていった！！
 > CONTEXT: Map132/events/1/pages/0/93/Dialogue
 For just an instant, I could make out Hyakki's figure in
-the bright flashes. But my vision was blocked up just as
-quickly as I was wrapped up in Titanoboa's coils!!#RonLocalization
+the bright flashes. But she was immediately
+wrapped up in Titanoboa's coils!!#RonLocalizationStrateRevision
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map132.txt:131]: Note

File: Gaijinizer patch/Map037.txt [hunk 8]
```diff
@@ -474,57 +490,64 @@ So I'll kill you in one shot...#Ron
...
 
 > BEGIN STRING
 アンタに言われなくったって
 
 こんなとこ、とっとと帰ってやるんだから！
 > CONTEXT: Map037/events/4/pages/0/341/Dialogue/enemy2
-I couldn't tell you. That's why I'm going home!#MTLed
+I'm going home even if you hadn't told me!#Strate
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map037.txt:551]: Some revision for flow:
```
I was going to go home even if you hadn't
told me!
```

File: Gaijinizer patch/CommonEvents_33.txt [hunk 9]
```diff
@@ -69,8 +69,8 @@ Haah... Haaaaah...... ♡#Ron
 
 少しだけ改めようと思った。
 > CONTEXT: CommonEvents_33/111/Dialogue
-I just... had an thought.
-I wanted to change just a little.#Ron
+I... wanted to change that previous thought
+just a little.#RonStrateRevision
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_33.txt:74]: Note

File: Gaijinizer patch/Map107.txt [hunk 10]
```diff
@@ -1335,9 +1354,10 @@ desire" is strong ...#MTLed
...
 \\N[1]のこと、『独占対象』だと思ってる。
 > CONTEXT: Map107/events/9/pages/1/177/Dialogue/hyakki
-I do not know well, but do you usually do it?
-Seeing a series of interactions, the other Echos
-I think that \\N[1] is the "exclusive object".#MTLed
+I'm not too sure, but I think this is not
+something she usually does? Seeing that series of
+exchanges between you two, the other Echos will
+also want to "monopolize" you, \\N[1].#StrateUnsure
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map107.txt:1361]: Note

File: Gaijinizer patch/Map107.txt [hunk 11]
```diff
@@ -1324,9 +1342,10 @@ Her body has transformed into a horrible figure ...! !#MTLed
...
  いかんせん、ちょいと『独占欲』が強くてサ……
 > CONTEXT: Map107/events/9/pages/1/164/Dialogue/hyakki
-Her name is "Echo". It's usually kind and gentle
-person. I'm sorry, a little bit "exclusive
-desire" is strong ...#MTLed
+Her name is "Echo".
+She's usually a kind and gentle girl, but 
+unfortunately her "desire to monopolize" is a
+little strong......#Strate
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map107.txt:1349]: Note

File: Gaijinizer patch/Map107.txt [hunk 12]
```diff
@@ -596,7 +602,9 @@ her face...#Ron
 ふらついてるぞ、\\N[1]。
 > CONTEXT: Map107/events/86/pages/0/2/Dialogue/other2
 > CONTEXT: Map107/events/7/pages/0/2/Dialogue/other2
-Oh yeah...I'm stuck, \\N[1].#MTLed
+H-hey......
+
+You're wobbling, \\N[1].#StrateTemporary
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map107.txt:608]: A tiny revision here:
```
H-hey...

You look wobbly there, \\N[1].
```

File: Gaijinizer patch/Map107.txt [hunk 13]
```diff
@@ -605,8 +613,9 @@ Oh yeah...I'm stuck, \\N[1].#MTLed
 ただちょっと……混乱しちゃって……
 > CONTEXT: Map107/events/86/pages/0/6/Dialogue/kazuya
 > CONTEXT: Map107/events/7/pages/0/6/Dialogue/kazuya
-I'm fine, I'm not tired...Just a little...I'm
-confused ...#MTLed
+I'm fine, I'm not particularly tired...
+
+Just a little...... confused......#StrateTemporary
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map107.txt:619]: "particularly" -> "that". Lower number of ellipses on each to 3.

File: Gaijinizer patch/Map107.txt [hunk 14]
```diff
@@ -615,8 +624,10 @@ confused ...#MTLed
 テントもある、少し休んでいけよ。
 > CONTEXT: Map107/events/86/pages/0/10/Dialogue/other2
 > CONTEXT: Map107/events/7/pages/0/10/Dialogue/other2
-It may be attacked again someday ...
-There is a tent, so take a break.#MTLed
+But you don't know when you'll be attacked
+again......
+
+There's a tent here, let's take a break.#StrateTemporary
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map107.txt:631]: Some revision here, too:
```
But you don't know when you'll be attacked
again...

There's a tent here, you should take a break.
```

File: Gaijinizer patch/Map107.txt [hunk 15]
```diff
@@ -1260,62 +1271,69 @@ storage box, too!#Ron
 
 また私たちを殺す気なんでしょう……！
 > CONTEXT: Map107/events/9/pages/1/62/Dialogue/enemy13
-..... I can not let you go, \\N[1].
-You will kill us again ...!#MTLed
+......I cannot let you go, \\N[1].
+
+You might kill us again......!#Strate
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map107.txt:1277]: Bottom set of ellipses to 3.

File: Gaijinizer patch/Map107.txt [hunk 16]
```diff
@@ -1260,62 +1271,69 @@ storage box, too!#Ron
...
 私たちは『小石』なんかじゃない！！
 
 あなたになんか……好きにさせたりしないから…！！
 > CONTEXT: Map107/events/9/pages/1/95/Dialogue/enemy13
-We are not pebble!!I don't want you to make me
-love you ...! !#MTLed
+We are not "pebbles"!!
+
+You won't make me fall in love...... with someone
+like you...!!#StrateUnsure
```
Comment (Triketos) [inline Gaijinizer patch/Map107.txt:1317]: Ron Note

**Review by Triketos — APPROVED** [review]
Looks good after you finish the last comments.

File: Gaijinizer patch/Items.txt [hunk 17]
```diff
@@ -897,6 +904,62 @@ human princess. If you give it to a specific demon...#Triketos
...
+Proof of Accomplishment#Strate
+> END STRING
+
+> BEGIN STRING
+宝石の魔力を凝縮した、エネルギー爆弾。
+属性無視の５００程度の魔法ダメージを与える
+> CONTEXT: Items/94/description
+An energy bomb containing the consensed magic power of gems.
+Deals about 500 damage, ignoring attributes.#StrateUnsure
+> END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Items.txt:941]: I assume "consensed" is "condensed"?

File: Gaijinizer patch/Items.txt [hunk 18]
```diff
@@ -897,6 +904,62 @@ human princess. If you give it to a specific demon...#Triketos
 The Tengu Prince#Triketos
 > END STRING
 
+> BEGIN STRING
+崇高を意味する白色の宝石。売ってもお金になる。
+\\C[14]妖魔に対して\\C[0]プレゼントすると、友好度が上がる。
+> CONTEXT: Items/91/description
+A white jewel symbolizing nobleness. Can be sold for money.
+\\C[14]Increases friendship when given to a demon\\C[0].#Strate
+> END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Items.txt:913]: Even though it says white in the original text, we want this to be "blue".

File: Gaijinizer patch/Items.txt (hunk 17 above)
Comment (Strate-S) [reply Gaijinizer patch/Items.txt]: fixed

File: Gaijinizer patch/Items.txt (hunk 18 above)
Comment (Strate-S) [reply Gaijinizer patch/Items.txt]: fixed

**Pull request merged**